src/utils/utils.js

```diff
@@ -0,0 +1,82 @@
+//select all from a table
+const selectAll = (aTable) => `SELECT * FROM ${aTable}`;
+
+//select all roles
+const selectAllRoles = (aTable, columns, aGroup) =>
+  `SELECT ${columns} FROM ${aTable} GROUP BY ${aGroup}`;
+
+//select all from a table
+const selectAllEmployees =
+  () => `SELECT e.id, e.first_name, e.last_name, r.title, d.name as department
+FROM employee e 
+JOIN employee e1 ON e1.id= e.id
+JOIN role r ON  e.role_id = r.id
+JOIN department d ON r.department_id = d.id`;
+
+//select by department
+const selectDepartmentEmployees = (aDepartment) => `SELECT * 
+FROM employee e, department d, role r
+WHERE e.role_id = r.id 
+and  r.department_id = d.id
+and d.name = '${aDepartment}'`;
+
+// select by manager
+const selectManagerEmployees = (managerId) => `SELECT * 
+FROM employee_db.employee e, employee_db.role r
+WHERE e.role_id = r.id 
+and e.manager_id = '${managerId}'`;
+//select department budget
+const selectDepartmentBudget = (aDepartment) => `SELECT sum(salary)
+FROM employee e, department d, role r
+WHERE e.role_id = r.id 
+and  r.department_id = d.id
+and d.name = '${aDepartment}'`;
+
+//add department
+const addAdepartment = (anId, aName) =>
+  `INSERT INTO department (id, name) VALUES ('${anId}', '${aName}')`;
+
+// get max id
+const maxId = (aDepartment) => `SELECT max(id) as max FROM ${aDepartment}`;
+
+//add role
+const addArole = (anId, aTitle, aSalary, aDepartment_id) =>
```

Comment:
Same as above, you don't need `an` and `a` at the beginning of the variable names. It is much easier to read if they are just `id`, `title`, etc.

Reply:
Done, thank you